webapp/vite.config.ts

```diff
@@ -14,8 +14,8 @@ export default defineConfig({
   },
   define: {
     'import.meta.env.APP_URL': JSON.stringify(process.env.APP_URL || ''),
-    'import.meta.env.DEGEN_CONTRACT_ADDRESS': JSON.stringify(process.env.DEGEN_CONTRACT_ADDRESS || '0xd4768df803c5a9eDA475159cfbBcF9c06F077c13'),
-    'import.meta.env.RESULTS_CONTRACT_ADDRESS': JSON.stringify(
+    'import.meta.env.VOCDONI_COMMUNITYHUBADDRESS': JSON.stringify(process.env.DEGEN_CONTRACT_ADDRESS || '0xd4768df803c5a9eDA475159cfbBcF9c06F077c13'),
+    'import.meta.env.VOCDONI_COMMUNITYRESULTSADDRESS': JSON.stringify(
```

Comment:
same

Reply:
and populate the changes in the files that import `import.meta.env.DEGEN_CONTRACT_ADDRESS` and `import.meta.env.RESULTS_CONTRACT_ADDRESS` (old names)

Reply:
Done